### DISCUSSION TITLE
Select with many items doesn't function correctly

### DISCUSSION THREAD

**Discussion by dannylin108** [chakra-ui/ark#2846] [General]
### Description

There're two problems if `<Select/>` has many items. 

1. If items don't fit top or bottom space from the trigger I have to use positioning to let it overlap the trigger button. But in this configuration it's merely not functioning: can't choose an item, since it closes immediately. 
2. If items don't fit viewport, they just get cut and there's no possibility to scroll to reach them.

https://github.com/user-attachments/assets/947b329f-f12f-4825-ba06-9925012d8675

### Link to Reproduction (or Detailed Explanation)

https://stackblitz.com/~/github.com/dannylin108/arkui-bug-report

### Steps to Reproduce

1.  Wait till web preview is ready 
3. Try to select an item another which happened to be under the mouse
4. It's impossible 
5. But should be possible 😊

### Ark UI Version

3.10.0

### Framework

- [ ] React
- [X] Solid
- [ ] Vue

**Comment by segunadebayo** [comment]
In the next version of Ark, the select component will be click-based instead of a pointerdown/up interaction. This should fix a bunch of usability issues.

For now, I recommend setting a maximum height for the content. Along the lines of:

```jsx
<Dialog.Content style={{ "max-height": "min(var(--available-height, 300px), 300px)" }}>
```

**Reply by dannylin108** [reply]
Ok, thank you!

**Comment by segunadebayo** [comment]
Closing — large lists are handled via `scrollToIndexFn` and `positioning.fitViewport`.
